upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -548,6 +546,14 @@ func (c *ApplyClusterCmd) Run() error {
 	if err != nil {
 		return fmt.Errorf("error running tasks: %v", err)
 	}
+
+	if !dryRun {
+		err = precreateDNS(cluster, cloud)
+		if err != nil {
```

Comment:
If you have a free scope block, you might as well `err := ` here. For bonus points, `if err := foo; err != nil {}` (but I recognize it's not prevailing `kops` style)